pkg/api/validation/validation_test.go

```diff
@@ -6431,3 +6431,41 @@ func TestIsValidSysctlName(t *testing.T) {
 		}
 	}
 }
+
+func TestWhitelistedSysctls(t *testing.T) {
+	valid := []string{
+		"net.foo.bar",
+		"kernel.shmmax",
+	}
+	invalid := []string{
+		"net.foo.bar",
+		"kernel.shmmax",
```

Comment:
That's intentional. The whole list is invalid because 2 of them are. We check for the two errors below.